<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginFrame</class>
 <widget class="QDialog" name="loginFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1941</width>
    <height>1039</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1941</width>
     <height>1041</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>login and create.png</pixmap>
   </property>
  </widget>
  <widget class="QFrame" name="rightFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>1041</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background-color: rgb(255, 255, 255);

}


</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="welcomeLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>1011</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font: 46pt &quot;Bebas&quot;;
	color: rgb(25, 25, 26);
}
</string>
    </property>
    <property name="text">
     <string>CCS Quequing System</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPassword1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>630</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid gray;
font: 14pt &quot;Montserrat&quot;;
border-radius: 5px;</string>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputEmail1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>550</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid gray;
font: 14pt &quot;Montserrat&quot;;
border-radius: 5px;</string>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>720</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 74, 173);
	color: rgb(255, 255, 255);
font: 14pt &quot;Montserrat&quot;;
border-radius: 20px;
}

</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGuest">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>790</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(254, 206, 47);
	color: rgb(0, 74, 173);
font: 14pt &quot;Montserrat&quot;;
border-radius: 20px;
}
</string>
    </property>
    <property name="text">
     <string>Guest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegister1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>790</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(0, 74, 173);
color: rgb(255, 255, 255);
font: 14pt &quot;Montserrat&quot;;
border-radius: 20px;
}
</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>711</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logoQueuing.png</pixmap>
    </property>
   </widget>
   <zorder>inputPassword1</zorder>
   <zorder>inputEmail1</zorder>
   <zorder>btnLogin</zorder>
   <zorder>btnGuest</zorder>
   <zorder>btnRegister1</zorder>
   <zorder>label_2</zorder>
   <zorder>welcomeLabel</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>inputEmail1</tabstop>
  <tabstop>inputPassword1</tabstop>
  <tabstop>btnLogin</tabstop>
  <tabstop>btnRegister1</tabstop>
  <tabstop>btnGuest</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
